cmd/ipfs/daemon.go

```diff
@@ -208,6 +209,13 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	// let the user know we're going.
 	fmt.Printf("Initializing daemon...\n")
 
+	if level := os.Getenv("BS_LOG"); level != "" {
+		logging.SetAllLoggers(logging.LevelWarn)
+		logging.SetLogLevel("bitswap", level)
+		logging.SetLogLevel("engine", level)
```

Comment:
The engine is too verbose so we might want to skip this and only track the other two.

Reply:
Most of the added logs though are in that component for the `INFO` level so we might want to move them to the more general BS or create a new log subsystem.